wgpu/src/backend/direct.rs

```diff
@@ -10,6 +10,8 @@ use crate::{
 use arrayvec::ArrayVec;
 use parking_lot::Mutex;
 use smallvec::SmallVec;
+#[cfg(any(feature = "glsl", feature = "spirv"))]
```

Comment:
Lets fully qualify all uses instead of Cow here.

Reply:
Done!